modules/export/mongoDb.js

```diff
@@ -121,6 +121,12 @@ const mongodbFormatItem = (parents, item, config, meta, _not = false) => {
   const children = item.get("children1");
   const {fieldSeparator, canShortMongoQuery} = config.settings;
 
+  const hasParentRuleGroup = parents.filter(it => it.get("type") == "rule_group").length > 0;
+  const parentPath = parents
+    .filter(it => it.get("type") == "rule_group")
+    .map(it => it.get("properties").get("field"))
+    .slice(-1).pop();
```

Comment:
No need to join fields as they are already full-pathed. Just get the last one (slice+pop will do this)